junit5-engine/src/main/java/org/junit/gen5/engine/junit5/descriptor/MethodTestDescriptor.java

```diff
@@ -175,17 +175,17 @@ private void invokeBeforeTestMethodCallbacks(ExtensionRegistry registry, TestExt
 				.forEach(extension -> executeAndMaskThrowable(() -> extension.beforeTestMethod(context)));
 	}
 
-	private void invokeTestMethod(ExtensionRegistry ExtensionRegistry, TestExtensionContext testExtensionContext,
+	protected void invokeTestMethod(JUnit5EngineExecutionContext context, TestExtensionContext testExtensionContext,
```

Comment:
Ah, never mind: It was changed so `TestFactoryTestDescriptor` can override the method.